web/cypress/support/page_objects/onboardingPage.ts

```diff
@@ -179,7 +179,9 @@ export class OnboardingPage {
   }
 
   selectLegalStructure(companyType: string) {
-    this.getLegalStructure(companyType).click({ force: true });
+    this.getLegalStructure(companyType).scrollIntoView();
+    this.getLegalStructure(companyType).check();
+    this.getLegalStructure(companyType).check();
```

Comment:
do we need this function, if we never interact with legal structure on the onboarding page anymore?